data-model/SDP-sample-records/work_sample1.json

```diff
@@ -0,0 +1,32 @@
+{
+  "id": "https://sinaimanuscripts.library.ucla.edu/work/10",
+  "label": "Hexaemeron (Basil of Caesarea)",
+  "uniform_title": "Basil, Saint, Bishop of Caesarea, approximately 329-379. Hexaemeron",
```

Comment:
Should this be "uniform_name" even in work records? That way there's only one field for work, person, place records that is our general 'headword'?